extract_cookies.py

```diff
@@ -60,7 +60,12 @@ def find_cookie_dir():
     ffdir = Path.home().joinpath(".mozilla/firefox/")
     paths = sorted(Path(ffdir).iterdir(), key=os.path.getatime)
 
-    return paths[-1].joinpath("cookies.sqlite")
+    # Add some smarter filtering to make sure the directory we're requesting
+    # is a profile and not some junk directory.
+    valid_paths = [candidate for candidate in paths 
+        if candidate.joinpath(Path("cookies.sqlite")).exists()]
```

Comment:
Thanks for the contribution.

According to the docs it isn't necessary to instantiate a Path() class object for joinpath, in their examples anyway, but it's fine and it's clear what it's doing.

Can you reformat the code so it will pass flake8? I haven't gotten around to adding any CI to this repo.

Something like:
```
    valid_paths = [
        candidate
        for candidate in paths
        if candidate.joinpath(Path("cookies.sqlite")).exists()
    ]
```
This will also fix a trailing whitespace on line 65.

Reply:
You got it. Updated to clear flake8 default checks.

Reply:
Awesome, thanks so much.